src/common/setup.ts

```diff
@@ -23,6 +23,14 @@ let otomiK8sVersion: string
 const checkKubeContext = async (): Promise<void> => {
   if (env.CI) return
   const debug = terminal('checkKubeContext')
+
+  if (existsSync('/var/run/secrets/kubernetes.io/serviceaccount/token')) {
```

Comment:
why have two checks (the first on line 24)?

Reply:
I was not sure about CI flag.
It skips checks of .secrets file
It also used to skip generation of sops file.

Reply:
> It skips checks of .secrets file

that is intended

> It also used to skip generation of sops file.

that is not the case anymore

you may close this PR as far as I am concerned ;)